docs/src/modules/spring/pages/actions-publishing-subscribing.adoc

```diff
@@ -1 +1,100 @@
+
+The following example assumes the existence of an xref:eventsourced.adoc[Event Sourced Entity] counter that emits events of type `CounterEvent` with subtypes `ValueIncreased` and `ValueDecreased`.
+
+[source,java,indent=0]
+.src/main/java/com/example/actions/CounterJournalToTopicAction.java
+----
+include::example$spring-eventsourced-counter/src/main/java/com/example/actions/CounterJournalToTopicAction.java[tag=sub-ESE-action]
+----
+<1> subscribing to the events from the Counter.
+<2> setting the method input type to the events produced by the counter.
+<3> any return is valid. See NOTE.
+
+The events from the entity are delivered to the Action. The implementation may vary, for this simplified example you are just logging it, but it could a forward to some other component or external service.
+
+[NOTE]
+====
+The return value of the method is an `Action.Effect` with message `0`, but can be any other of type `Action.Effect<Integer>` as the return type of the method indicates. The Kalix framework needs the response to ensure that the event was successfully processed. If no exception is thrown and the method returns a value, the framework assumes that the event was successfully processed and marks it as such. This allows the next event to be sent to the subscribing method.  However, if an exception is raised and not handled, this action will not process any more events until the necessary handling is added. It will raise the same error over and over again until the application is fixed and restarted.
+====
+
+
+== Subscribing to a Value Entity
+
+You can subscribe an Action to events from an Value Entity. To receive messages from the entity, annotate a service method `@Subscribe.ValueEntiy` and specify the class of the entity. The same applies as for subscribing to an Event Sourced Entity.
+
+
+== Subscribing to a Topic
+
+To receive messages from a from Google Cloud Pub/Sub or Apache Kafka topic, annotate the service method `@Subscribe.Topic` and specify the topic name.
+
+The events from the topic are delivered to the Action and in this simplified example they are only logged.
+
+[source,java,indent=0]
+.src/main/java/com/example/actions/CounterJournalToTopicAction.java
+----
+include::example$spring-eventsourced-counter/src/main/java/com/example/actions/CounterJournalToTopicAction.java[tag=sub-topic-action]
+----
+<1> subscribing to the events from topic 'counter-events'. 
+<2> setting the method input type to the events produced by the counter.
+<3> any return is valid. See NOTE in Subscribing to an Event Sourced Entity.
+
+
+Note that by default, Kalix assumes the message in the topic was serialized as JSON and as such deserialize it into a `CounterEvent`. You can find more about serialization in Kalix here xref:spring:serialization.adoc[]
```

Comment:
Make use of the "NOTE" special section

```suggestion
NOTE: by default, Kalix assumes the message in the topic was serialized as JSON and as such deserialize it into a `CounterEvent`. You can find more about serialization in Kalix here xref:spring:serialization.adoc[].
```

Reply:
I used it already I was feeling I would be abusing it.

Reply:
I haven't actually rendered it, so you certainly have a better perception..

Reply:
I used your suggested below "IMPORTANT".